pages.th/linux/clamav.md

```diff
@@ -0,0 +1,21 @@
+# clamav
```

Comment:
Oh, the English page is not valid. This should be an alias page, or similar to https://github.com/tldr-pages/tldr/blob/main/pages/common/transmission.md

Reply:
Oh ok .. I just saw that too.  I just changed the original English content of clamav to be an alias of clamscan and adjusted the Thai page of clamav accordingly.

Reply:
:thinking: Instead of the edition of the `clamav` page, translate `clamscan`. Let's work on `clamav` in another PR.

Reply:
Certainly.  This is done.  I have also removed clamav's Thai translation.